scripts/seed-dev.sh

```
@@ -225,6 +246,138 @@ echo "== seed-dev: demo organization =="
+# story the demo tells.
+employ() { # employ <full-name> <role>
+  local name="$1" role="$2" id edges standing rel_id rel_version status
+  id="$(person_id "$name")"
+  [ -n "$id" ] || fail "$name is not in the installation the seed just wrote to"
+  edges="/relationships?kind=employment&person_id=$id"
+  if [ -n "$(find_first "$edges" ".organization_id == \$org and $CURRENT_PRIMARY_JQ" --arg org "$org_id")" ]; then
+    echo "  OK: $name already employed at Demo GmbH"
+    return
+  fi
+  standing="$(find_first "$edges" \
+    '.organization_id == $org and (.ended_at == null or (.ended_at | tostring) >= $today)' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)